### ISSUE TITLE
[Compatibility Request] Extended Native Syntax could play nicer with Advanced Tables plugin

### ISSUE THREAD

**Issue by huyz** [aidenlx/table-extended#21]
I'm trying to use both this plugin using Extended Native Syntax and another plugin https://github.com/tgrosinger/advanced-tables-obsidian, which pretty formats the table when you hit `tab` or `shift`+`tab`

- `^^` works
- left/center/right justification works

The following don't work:
- for colspan, the `||` is incorrectly split apart by Advanced Tables into a column. So how about recognizing that `|  :  |` is equivalent to `||`? That way, Advanced Tables could be free to reformat the table and Table Extended could recognize my intent to colspan
- for multiline, there must be a `\` at the end of the line, outside the table.  But upon reformatting, Advanced Tables creates a new right-most column with the `\`s in it.  So how about recognizing an empty column with only `\` inside of it as meaning the same thing?

**Comment by aidenlx** [comment]
here is the status: https://github.com/tgrosinger/advanced-tables-obsidian/issues/59#issuecomment-812886995

**Comment by huyz** [comment]
Yes it would be cleaner if Advanced Tables improved compatibility, as I suggested in this ticket too: https://github.com/tgrosinger/advanced-tables-obsidian/issues/119

But who knows when they will implement https://github.com/tgrosinger/advanced-tables-obsidian/issues/59#issuecomment-812886995 ? 
May be years.

So in the meantime, the workarounds I suggested above would hold us over while we wait. Yes, not as good, but better than waiting forever 😄
